core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -251,6 +335,7 @@ package object config {
   private[spark] val FILES_MAX_PARTITION_BYTES = ConfigBuilder("spark.files.maxPartitionBytes")
     .doc("The maximum number of bytes to pack into a single partition when reading files.")
     .longConf
+    .checkValue(_ > 0, "The maximum number of bytes event queue must not be negative")
```

Comment:
Since `_ > 0`, we should avoid 0, too.
`must not be negative` => `must be positive`.